requirements/pip.txt

```diff
@@ -4,11 +4,11 @@
 #
 #    make upgrade
 #
-wheel==0.37.1
+wheel==0.38.2
     # via -r requirements/pip.in
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==22.3
+pip==22.3.1
```

Comment:
No breaking change in new version: https://pip.pypa.io/en/stable/news/#bug-fixes